### DISCUSSION TITLE
Welcome to ScraGet Discussions!

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Quantum-Codes/ScraGet#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sid72020123** [comment]
Hi!

**Reply by Quantum-Codes** [reply]
Hiii!

**Comment by G8MR** [comment]
Hey there!

**Reply by Sid72020123** [reply]
Hi!

**Reply by Quantum-Codes** [reply]
> Hey there!

Hi!!

**Comment by pikachub2005** [comment]
People still make these? ;-; reminds me of the old days

**Reply by Quantum-Codes** [reply]
Uh... make what?
